activities/communication/blogging.md

```diff
@@ -52,34 +48,48 @@ After writing, please share with the communications coordinator for polishing.
 
 ## How to create a blogpost on the Foundation’s blog
 
-1. Create a branch for your new blogpost within in the [blog repo](https://github.com/publiccodenet/blog).
+### Step 1: create your branch and file
+
+Create a branch for your new blogpost within in the [blog repo](https://github.com/publiccodenet/blog).
+
+Create a new file for your blogpost within [`_posts/` on the blog repository](https://github.com/publiccodenet/blog/tree/develop/_posts)
+named `YEAR-MONTH-DAY-title.md`.
```

Comment:
I m not sure, but I believe the most clear way to avoid confusion is to ask for the format YYYY-MM-DD-title.md (to avoid someone writing FEBRUARY in the filename).

Reply:
Done!